pkg/render/policyrecommendation.go

```diff
@@ -381,6 +384,12 @@ func (pr *policyRecommendationComponent) deployment() *appsv1.Deployment {
 			Template: *podTemplateSpec,
 		},
 	}
+
+	if overrides := pr.cfg.PolicyRecommendation; overrides != nil {
```

Comment:
Same here - shouldn't this be `pr.cfg.PolicyRecommendation.PolicyRecommendationDeployment`?